.jscsrc

```diff
@@ -0,0 +1,33 @@
+{
+    "requireCamelCaseOrUpperCaseIdentifiers": true,
+    "requireCapitalizedConstructors": true,
+    "requireCommaBeforeLineBreak": true,
+    "requireSemicolons": true,
+
+    // spaces
+    "requireSpaceBeforeBlockStatements": true,
+    "requireSpaceBeforeKeywords": [
+        "else",
+        "while",
+        "catch"
+    ],
+    "requireSpaceAfterKeywords": true,
+    "requireSpaceBeforeObjectValues": true,
+    "requireSpaceBetweenArguments": true,
+    "requireSpacesInConditionalExpression": true,
+    "requireSpacesInFunction": {
+        "beforeOpeningRoundBrace": true,
```

Comment:
Set this to `false` please! In fact, I'd like to make sure there *isn't* a space before the opening round brace - will "false" do that?

That's just my style :)